recipes/exchcxx/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "exchcxx" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wavefunction91/ExchCXX/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0ea38aab563ea5d11a03d80dc1bf909821091d903eb852c5cb350484753a847a
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja
+  host:
+    - libxc-c >=7,<8
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libexchcxx$SHLIB_EXT  # [unix]
+    - test -f $PREFIX/include/exchcxx/xc_kernel.hpp  # [unix]
+    - test -f $PREFIX/lib/cmake/ExchCXX/ExchCXXConfig.cmake  # [unix]
```

Comment:
Not necessary, but if you want to also check the existance of a given target you can also use https://github.com/ami-iit/cmake-package-check, see https://github.com/search?q=org%3Aconda-forge%20cmake-package-check&type=code .